### DISCUSSION TITLE
Portable pump and energy

### DISCUSSION THREAD

**Discussion by Gobrich** [FakeFishGames/Barotrauma#10666] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The "Pump" parameter does not work for a Portable pump. I set the Min voltage and Power Consumption as 0 and 0 respectively, but the p.pump consumed the same amount of energy as with standard parameters. This is how it works with any exposed numbers.
I will note that I am not playing a campaign, but only building boats in your editor. I would really like to introduce a function with "battery cell"-powered pumps, but this is not possible now because of too fast energy consumption. I'm asking you to do something about it.
UPD: While I was writing the text, I noticed that Power Consumption and Min Voltage do not work on all items that use battery cell to work.

### Bug prevalence

Happens regularly

### Version

0.19.14.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Singularia** [comment]
If i understand correctly the OP suggests giving sub-builders the ability to edit how much, or how fast some items (in his case Portable pump) would consume their "power" source (like Battery cells, O2/Welding tanks/Fuel rods etc.)

**Reply by Gobrich** [reply]
That's right
